content/security/advisory/2017-11-08.adoc

```diff
@@ -1,14 +1,34 @@
 ---
-layout: simplepage
+layout: advisory
 title: Jenkins Security Advisory 2017-11-08
 section: security
 kind: core
+core:
```

Comment:
Probably not really great to add special treatment for core but given how advisories typically look it's probably a worse idea to inline this entire part for each of the fixed issues.

If we ever want complete information about whether an issue affects core or not, we can always add a flag to its entry.